test/unit/helpers/form_helper_test.rb

```diff
@@ -51,4 +51,8 @@ class FormHelperTest < ActionView::TestCase
       end
     end
   end
+
+  def h(input)
```

Comment:
I've added a comment and put the same method to other test that was failing on the same error